docs/components/listbox/use-cases.md

```diff
@@ -87,61 +84,83 @@ This behavior can usually be seen in `<select>` on the Windows platform.
+  <lion-option .choiceValue=${'Cabbage'}>Cabbage</lion-option>
+  <lion-option .choiceValue=${'Carrot'}>Carrot</lion-option>
+</lion-listbox>
 ```
 
 ## Rotate keyboard navigation
 
 `rotate-keyboard-navigation` attribute on the listbox will give the first option active state when navigated to the next option from last option.
 
-```js preview-story
-export const rotateKeyboardNavigation = () => html`
-  <lion-listbox name="combo" label="Rotate keyboard navigation" rotate-keyboard-navigation>
-    <lion-option .choiceValue=${'Apple'}>Apple</lion-option>
-    <lion-option .choiceValue=${'Artichoke'}>Artichoke</lion-option>
-    <lion-option .choiceValue=${'Asparagus'}>Asparagus</lion-option>
-    <lion-option .choiceValue=${'Banana'}>Banana</lion-option>
-    <lion-option .choiceValue=${'Beets'}>Beets</lion-option>
-    <lion-option .choiceValue=${'Bell pepper'}>Bell pepper</lion-option>
-    <lion-option .choiceValue=${'Broccoli'}>Broccoli</lion-option>
-    <lion-option .choiceValue=${'Brussels sprout'}>Brussels sprout</lion-option>
-    <lion-option .choiceValue=${'Cabbage'}>Cabbage</lion-option>
-    <lion-option .choiceValue=${'Carrot'}>Carrot</lion-option>
-  </lion-listbox>
-`;
+```html preview-story
+<lion-listbox name="combo" label="Rotate keyboard navigation" rotate-keyboard-navigation>
+  <lion-option .choiceValue=${'Apple'}>Apple</lion-option>
+  <lion-option .choiceValue=${'Artichoke'}>Artichoke</lion-option>
+  <lion-option .choiceValue=${'Asparagus'}>Asparagus</lion-option>
+  <lion-option .choiceValue=${'Banana'}>Banana</lion-option>
+  <lion-option .choiceValue=${'Beets'}>Beets</lion-option>
+  <lion-option .choiceValue=${'Bell pepper'}>Bell pepper</lion-option>
+  <lion-option .choiceValue=${'Broccoli'}>Broccoli</lion-option>
+  <lion-option .choiceValue=${'Brussels sprout'}>Brussels sprout</lion-option>
+  <lion-option .choiceValue=${'Cabbage'}>Cabbage</lion-option>
+  <lion-option .choiceValue=${'Carrot'}>Carrot</lion-option>
+</lion-listbox>
 ```
 
 ## Disabled options
 
 Navigation will skip over disabled options. Let's disable Artichoke and Brussel sprout, because they're gross.
 
+```html preview-story
+<lion-listbox name="combo" label="Rotate with disabled options" rotate-keyboard-navigation>
+  <lion-option .choiceValue=${'Apple'}>Apple</lion-option>
+  <lion-option .choiceValue=${'Artichoke'} disabled>Artichoke</lion-option>
+  <lion-option .choiceValue=${'Asparagus'}>Asparagus</lion-option>
+  <lion-option .choiceValue=${'Banana'}>Banana</lion-option>
+  <lion-option .choiceValue=${'Beets'}>Beets</lion-option>
+  <lion-option .choiceValue=${'Bell pepper'}>Bell pepper</lion-option>
+  <lion-option .choiceValue=${'Broccoli'}>Broccoli</lion-option>
+  <lion-option .choiceValue=${'Brussels sprout'} disabled>Brussels sprout</lion-option>
+  <lion-option .choiceValue=${'Cabbage'}>Cabbage</lion-option>
+  <lion-option .choiceValue=${'Carrot'}>Carrot</lion-option>
+</lion-listbox>
+```
+
+## Validation
```

Comment:
Maybe we should mention here that it inherits from LionField/FormControl and validation works the same for all FormControls (the less code examples we need to maintain, the better)

For Combobox the example made sense since it has a specific Validator